packages/devtools/lib/src/framework/framework.dart

```diff
@@ -496,6 +500,7 @@ abstract class Screen {
     this.disabledTooltip = 'This screen is not available',
     bool disabled = false,
     this.shortcutCallback,
+    this.isPartOfMainNavigation = true,
```

Comment:
isPartOfMainNavigation is a bit long. How about
nit: isPartOfMainNavigation --> showTab

Reply:
ok, updated